test/node_smoke_tests/commonjs/document_missing.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
```

Comment:
The previous node smoke tests have been moved to the `commonjs` folder (with some modifications), their ESM versions are in the `module` folder.